recipe/meta.json

```diff
@@ -4,27 +4,27 @@
   "name": "uwtools",
   "packages": {
     "dev": [
-      "black =23.3.*",
-      "coverage =7.2.*",
+      "black =23.11.*",
+      "coverage =7.3.*",
```

Comment:
Your call,, but I also added a TODO item in [UW-436](https://jira.epic.oarcloud.noaa.gov/browse/UW-436) to update the versions in the docs when `doc-update` eventually merges to `develop`. Since we don't plan on merging `develop` into `doc-updates`, I think we'll get this taken care of at the right time.